components/elm/src/cpl/lnd_import_export.F90

```diff
@@ -1446,7 +1446,8 @@ subroutine lnd_export( bounds, lnd2atm_vars, lnd2glc_vars, l2x)
        endif
        l2x(index_l2x_Flrl_Tqsur,i)  = lnd2atm_vars%Tqsur_grc(g)
        l2x(index_l2x_Flrl_Tqsub,i)  = lnd2atm_vars%Tqsub_grc(g)
-       l2x(index_l2x_coszen_str,i) = lnd2atm_vars%coszen_str(g)
+       l2x(index_l2x_coszen_str,i)  = lnd2atm_vars%coszen_str(g)
+       l2x(index_l2x_Flrl_rofmud,i) = lnd2atm_vars%qflx_rofmud_grc(g)
```

Comment:
What will happen if soil erosion is turned off which is the default option? Will this introduce weird behavior to the model?

Reply:
The current test is for data-land mode only, i.e., MOSART-sediment will read rofmud from existing inputs, so there won't be weird model behavior if soil_erosion is turned off. Later, in a separate PR perhaps, when we add a test for coupled ELM-soil erosion and MOSART-sediment, we want to make sure once sediflag is on, soil_erosion is also on.